### ISSUE TITLE
automatic theming

### ISSUE THREAD

**Issue by wjakethompson** [OttaviaE/implicitMeasures#4]
for openjournals/joss-reviews#2394 

This is very minor and just a pet peeve of mine, so feel free to ignore if you have strong feelings, but is there a reason for overriding the default ggplot2 theme in the plotting functions?

When I am generating plots from packages such as this, I often have to apply my own themes to match the report template or specific journal standards, depending on the dissemination plan.  When the theme elements are changed within a function, it can make it difficult to figure out what all needs to be changed. I think this is especially true when removing axis titles. For example, if the function sets `theme(axis.title.x = element_blank())`, and I then try to modify the returned plot with `labs(x = "some title")`, it won't show up. For someone who isn't a ggplot2 aficionado, this could be very confusing.

In general, my preference is for a function to return the default ggplot2 object, and then the user can modify it as they see fit. If you opt to apply non-default themes, I would recommend consistency in the themes. Most use `theme_minimal()`, but it looks like `d_plot()` is using `theme_classic()`.

**Comment by OttaviaE** [comment]
Thank you for your suggestions! I actually changed a little bit the code, so now there is a consistency in the themes used (it's always `theme_minimal()`. Besides, I also allowed the users to rename the axis names by avoiding the setting `theme(axis.title.x = element_blank())`. I did not leave the deafult ggplot settings because sometimes it might be pretty difficult to use and understand and I wanted users to have pretty graphical representations with the least possible effort.
I hope that with the new changes, the graphs can be easily further modified.
